lib/ansible/modules/net_tools/nios/nios_a_record.py

```diff
@@ -116,6 +116,17 @@
       password: admin
   connection: local
 
+- name: update an A record IP address
+  nios_a_record:
+    name: a.ansible.com
+    ipv4: {new_ipv4addr: 192.168.11.10, old_ipv4addr: 192.168.10.1}
```

Comment:
you have added new params `new_ipv4addr` and `old_ipv4addr` for updating IP address but u haven't added the params in doc param. Also, since adding of new param is a new feature, so it's better to handle the scenario in different PR as the respective PR won't be backported to `2.9.x` but other bug fixes of this PR can be backported to `2.9.x`

Reply:
I added it here as a fix for #64045
Let me know if you need me to remove it.

Reply:
yes, u need to create a diff PR for this fix as this fix needs addition of new param and which is considered as new feature. Also, u need to add these two new params in module documentation (ref: [nios_host_record](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/net_tools/nios/nios_host_record.py#L32))

And for other fixes u can create a backport to 2.9.x branch

Reply:
I've implemented the requested change.